### ISSUE TITLE
Not updating path in file.

### ISSUE THREAD

**Issue by adamcarr** [smysnk/gulp-rev-all#138]
Previously, if I had a relative path string in a file that is not a require or define, like `var test = './folder/path';` it would update this. After your 3/3 release, this is no longer updates.

**Comment by adamcarr** [comment]
Now, even reverting back to 0.8.22, I still see this locally. Perhaps a dependency changed that would cause this?

**Comment by adamcarr** [comment]
Commit e7d945aad24cd55d970279225990866c4b0fbc29 breaks a relative file path regex for subfolder files. Given a relative path `./routes/dashboard/dashboard`, the regex created is `/('|")(\/routes\/dashboard\/dashboard)()('|"|$)/g` and `/([^a-zA-Z0-9\.\-\_\/])(\/routes\/dashboard\/dashboard)(\.js)([^a-zA-Z0-9\.\-\_\/]|$)/g` Which break. Reverting that commit fixes the issue.

**Comment by adamcarr** [comment]
Also, i have the above commit files in my node_modules folder for version 0.8.22 but if I look at the git tag for 0.8.22, those changes are not present as it is a snapshot from October, 2015.

**Comment by CesarD** [comment]
Is this issue related to something this? : http://stackoverflow.com/questions/39792419/gulp-rev-all-not-replacing-paths-correctly-in-html-located-in-subfolder

**Comment by smysnk** [comment]
@CesarD You are windows? What version of gulp-rev-all are you using? I have released a new version recently that includes a windows PR fix.

**Comment by CesarD** [comment]
Yes, I'm on windows. I was using the most recent version to that date. Due to project delivery times I had to go back to the combo gulp-rev & gulp-rev-replace and it worked with no issues.
I could try again tomorrow with the latest version of gulp-rev-all and see if it got fixed or not.
Thanks.
